### PR TITLE
subメソッドを追加

### PR DESCRIPTION
**Pull request by Tecco**

### CODE REVIEW

File: nobita.js [hunk 1]
```diff
@@ -8,6 +8,15 @@ function main() {
 	console.log("who?")
 }
 
+function sub() {
+	if (isNobita) {
+		console.log("suki")
+	} else {
+		console.log("kirai")
+	}
```
Comment (nazcadon) [inline nobita.js]:
```
function sub() {
	if (isNobita) {
		console.log("suki")
                console.log("doraemon")
       }
	return
		console.log("kirai")
                 console.log("doraemon")
```

File: nobita.js [hunk 2]
```diff
@@ -8,6 +8,15 @@ function main() {
 	console.log("who?")
 }
 
+function sub() {
```
Comment (newt0) [inline nobita.js]:
```
function sub() {
	if (isNobita) {
		return "suki" + "doraemon
	} 
    console.log( "kirai" + "doraemon)
}



```

**Comment by nazcadon** [comment]
LGMT

**Comment by newt0** [comment]
LGTM

**Comment by newt0** [comment]
https://i.pinimg.com/564x/f3/7c/9e/f37c9e58b9e08d621c68d0df81a14564.jpg

**Pull request merged**